frontend/src/components/CreateEntryModal.svelte

```diff
@@ -108,6 +108,22 @@
                     console.log(error);
                 } else {
                     reviewers.map(reviewer => createDirectRequest(reviewer.id))
+                    if (open_slots !== 0) {
```

Comment:
not important, but why not write 
open_slots greater than zero

Reply:
doesn't make much of a difference since only values 0 or above are accepted anyways before the call is made